src/test/resources/hudson/tasks/junit/JENKINS-41134/TestSuite_second.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites>
+    <testsuite name="TestSuite_second" tests="1" errors="0" failures="1" skip="0" timestamp="2012-04-05T10:50:00">

```

Comment:
Are we worried about the case when the same exact test runs and has the same exact timestamp? If the two tests with the same name have a different timestamp, they are considered a different instance of the test. If they have the same name, and the same timestamp they are considered the same. What is the likelihood that the same test suite will run at the same exact time on two nodes?